20_2 TreeList Filter Panel: rewrite (#780)
DevExpress/devextreme-demos: JSDemos/Demos/TreeList/FilterPanel/description.md

> The [filterPanel](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterPanel/) is a UI element that displays the combined filter in the bottom of the TreeList. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterValue) property and consists of filters that users apply in the UI elements ([filter row](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterBuilder)). Users can click the checkbox in the filter panel to apply or deactivate the current filter.

Reviewer: I wouldn't use "UI element" the first time, so that it doesn't sounds that much as a "definition". The second time it's good, because we need a word that references all other ways to set a filter. All in all - a couple of minor changes.

Also I'm removing repetition of the word "filter" - you can use "conditions", "criteria"... 

```suggestion
The [filterPanel](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterPanel/) displays the combined filter in the bottom of the TreeList. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterValue) property and consists of conditions that a user applied in the following UI elements: [filter row](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterBuilder). Users can deselect the checkbox in the filter panel to temporarily deactivate the current filter.
```

> The [Filter panel](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/filterPanel/) is a UI element that displays the combined filter situated in the bottom of the TreeList. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterValue) property and consists of [custom filters](/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/customOperations) and filters that users apply in the UI elements ([filter row](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilder)).

Reviewer 2: AFAIK, custom filters are applied using the same UI elements.

```suggestion
The [filterPanel](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/filterPanel/) is a UI element that displays the combined filter in the bottom of the TreeList. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterValue) property and consists of filters that users apply in the UI elements ([filter row](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilder)). Users can click the checkbox in the filter panel to apply or deactivate the current filter.
```

To display the filter panel, set the **filterPanel**.[visible](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/filterPanel/#visible) property to **true**. 

> A click on the combined filter calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilder) object. You can also click the check box to toggle the current filter. The filter values are always reflected in the filter row.

Reviewer 2: > The filter values are always reflected in the filter row.

Not true. They are reflected in the header filter too.

> You can also click the check box to toggle the current filter.

Not you but users, and it bears no relation to the filter builder. I moved it to the first paragraph.

```suggestion
A click on the combined filter calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilder) object.
```

> A click on the combined filter calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilder) object. You can also click the check box to toggle the current filter. Note that filter values are always reflected in the filter row.

Reviewer 3: ```suggestion
A click on the combined filter calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilder) object. You can also click the check box to toggle the current filter. The filter values are always reflected in the filter row.
```

> DataGrid uses the DevExtreme [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) component to display the integrated filter builder. The popup's default configuration is defined automatically, but you can change it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilderPopup) object.

Reviewer 2: ```suggestion
DataGrid uses the DevExtreme [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) component to display the integrated filter builder. The Popup's default configuration is defined automatically, but you can change it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilderPopup) object.
```

> DataGrid uses the DevExtreme [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) component to display the integrated filter builder. The popup's default configuration is defined automatically, but you can change it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilderPopup) object.

Reviewer 2: Don't forget to replace DataGrid with TreeList.

> The intergrated filter builder is displayed in a pop-up window (the DevExtreme [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) component). Its default configuration is defined automatically, but you can change it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilderPopup) object.

Reviewer 3: ```suggestion
DataGrid uses the DevExtreme [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) component to display the integrated filter builder. The popup's default configuration is defined automatically, but you can change it in the [filterBuilderPopup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#filterBuilderPopup) object.
```
